iot_packet_verifier/docs/design.md

```diff
@@ -0,0 +1,81 @@
+# IoT Packet Verifier - Design
+
+See [README](../README.md) for overview.
+
+Data flow with AWS S3 elided:
+
+```
+HPR → PV → PS → CS
+ ↑              /
+  \____________/
+```
+
+where:
+
+- HPR is [Helium Packet Router](https://github.com/helium/helium-packet-router/)
+- PV is [this](..) component, the IoT Packet Verifier
+- PS is the Payment System
+  + follows the blockchain
+  + credits Rewards to gateway owners' wallets for providing the network
+  + debits Data Credits (DC) of OUI owners' wallets for use of the network
+- CS is the Config Server associated with HPR
+
+## Packet Verifier
+
+- Extract metadata from Protobufs via S3 (read-only)
+- Sum packet counters by gateway for Rewards
+- Sum packet counters by OUI for debiting DC
+- Write output as Protobufs via S3 (create & append only)
+- Fetch from Config Server:
+  + List of customer OUIs and their NetIDs
+
+## Payment Server
+
+- Consumes reports generated by PV
```

Comment:
Note that this is the iot rewards server. Don’t invent new names

Reply:
Addressed with commit e0e1c6a based upon conversation earlier today.